.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+# Contributing to `testcontainers-python`
+
+Welcome to the `testcontainers-python` community!
+This should give you an idea about how we build, test and release `testcontainers-python`!
+
+Highly recommended to read this document thoroughly to understand what we're working on right now
+and what our priorities are before you are trying to contribute something.
+
+This will greatly increase your chances of getting prompt replies as the maintainers are volunteers themselves.
+
+## Before you Begin
+
+We recommend following these steps:
+
+1. Finish readng this document.
+2. Read the [recently updated issues][1]
+3. Look for existing issues on the subject you are interested in - we do our best to label everything correctly
+
+
+## Local Development
+
+### Pre-Requisites
+
+You need to have the following tools available to you:
+- `make` - You'll need a GNU Make for common developer activities
+- `poetry` - This is the primary package manager for the project
+- `pyenv` **Recommended**: For installing python versions for your system.
+  Poetry infers the current latest version from what it can find on the `PATH` so you are still fine if you don't use `pyenv`.
+
+### Build and test
```

Comment:
I'm using the devcontainers workflow. I could add in a section on using that if you wanted.

Reply:
hey @bearrito apologies for the delays, I'm finding myself with some time today.
I'd love your contribution on the devcontainers workflow, but lemme finish this PR (ETA today, finally found the time to work on this) and then yo can raise your own PR for that addition to the docs :pray: